util/storage-samples/storage-owner-sample/src/test/java/org/fidoalliance/fdo/storage/To2StorageTest.java

```diff
@@ -156,8 +156,8 @@
   String packageName = "linux64.sh";
   String filename = "sample_file";
   String url = "http://host/file.tmp";
-  String sviString = "sdo_sys:filedesc=packageName,sdo_sys:write=packageContent" +
-      ",sdo_wget:filename=filename,sdo_wget:url=url";
+  String sviString = "fdo_sys:filedesc=packageName,fdo_sys:write=packageContent" +
```

Comment:
The test-cases referencing these are commented out in the source. They should be removed in a future commit.